packages/markdown/remark/src/index.ts

```diff
@@ -24,8 +24,8 @@ import { rehypeImages } from './rehype-images.js';
 export { InvalidAstroDataError, setVfileFrontmatter } from './frontmatter-injection.js';
 export { rehypeHeadingIds } from './rehype-collect-headings.js';
 export { remarkCollectImages } from './remark-collect-images.js';
-export { remarkPrism } from './remark-prism.js';
-export { remarkShiki } from './remark-shiki.js';
```

Comment:
I think we still need to preserve exporting `remarkPrism` and `remarkShiki` here to prevent a breaking change, and the potential where an older version of Astro or `@astrojs/mdx` uses this newer package version (somehow).


We can add a TODO to remove it in the next major of this package.